mongodb/mongo_common.h

```diff
@@ -0,0 +1,68 @@
+/**                                                                 -*- C++ -*-
+ * mongo_common.h
+ * Jeremy Barnes, 23 February 2015
+ * Mich, 2016-08-02
+ * This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
+ **/
+
+#pragma once
+#include "bsoncxx/builder/stream/document.hpp"
+
+#include "sql/sql_expression.h"
+#include "server/dataset_context.h"
+
+namespace Datacratic {
+
+struct Date;
+
+namespace MLDB {
+
+struct CellValue;
+struct PathElement;
+struct ExpressionValue;
+typedef std::vector<std::tuple<PathElement, ExpressionValue>> StructValue;
+
+namespace Mongo {
+
+const Package & mongodbPackage();
+
+const static std::string mongoScheme =
```

Comment:
This is more than a scheme.  We should also add the word description as I thought it was only the scheme.